### PR TITLE
Replace refs on clan CC lines

### PR DESCRIPTION
**Pull request by tgrego**
references on clan comments now get a reference replacement just like the pfam comments, so [1,2] becomes something like [PMID:111,PMID:222].

References in the form [1-3] were ignored, this also fixes that so they become something like [PMID:111,PMID:222,PMID:333]

### CODE REVIEW

**Review by matthiasblum — CHANGES_REQUESTED** [review]
Thanks. Minor changes required, please see comments/suggestions.

**Comment by tgrego** [comment]
updated based on comments/suggestions, thanks!

File: pyinterprod/interpro/contrib/pfam.py [hunk 1]
```diff
@@ -109,7 +109,21 @@ def _repl(match: re.Match) -> str:
...
+    for i in s.split(','):
+        values = i.split("-")
+        if len(values) == 1:
+            # single reference number
+            r.append(int(i))
+        else:
+            # range of reference numbers, e.g. 1-4
+            low, high = map(int, values)
+            r += list(range(low, high + 1))
+    return(r)
```
Comment (matthiasblum) [inline pyinterprod/interpro/contrib/pfam.py:126]: Blank line before `return`, and also `return` is a statement and doesn't need parentheses, unless you want to return a tuple.

```suggestion

    return r
```

**Review by matthiasblum — APPROVED** [review]
I made a minor change. Feel free to merge.

**Pull request merged**